cloudblue_connect.custom_adapter.rb

```diff
@@ -0,0 +1,694 @@
+            { name: "external_uid"}
+            ]
+          },
+          { name: "errors", type: "array"},
+          
+          
+         ]
+      end
+    },
+    request: {
+      fields: lambda do
+        [
+          { name: "id"},
+          { name: "type"},
+          { name: "note"},
+          { name: "reason"},
+          { name: "created"},
+          { name: "updated"},
+          { name: "answered""note"},
+          { name: "reason"},
+          { name: "assignee"},
+          { name: "activation_key"},
+          { name: "status" },
+          { name: "asset", type: "object", properties: [
+            { name: "id" },
+            { name: "status" },
+            { name: "external_id" },
+            { name: "external_uid" },
+            { name: "product", type: "object", properties: [
+              { name: "id" },
+              { name: "icon" },
+              { name: "name" },
+              { name: "status" }
+            ]},
+            # skipped connection and events structures
+            { name: "items", type: "array", properties: [
+              { name: "id" },
+              { name: "global_id" },
+              { name: "mpn" },
+              { name: "old_quantity" },
+              { name: "quantity" },
+              { name: "type" },
+              { name: "display_name" },
+              { name: "period" },
+              { name: "item_type" }
+            ]},
+            # skipped params array, tiers and configuration structures
+            { name: "marketplace", type: "object", properties: [
+              { name: "id" },
+              { name: "name" },
+              { name: "icon" }
+            ]}
+            # skipped contract structure
+          ]}
+        ]
+      end
+    },
+  },
+  methodes: {
+  },
+  actions: {
+
+
+    asset_cancel_request: {
+      title: "Create Asset Cancel Request",
+      subtitle: "",
+      description: "",
+      help: "<a href='https://connect.cloudblue.com/community/api/openapi/#operations-Assets-request_list_listRequests' target='_blank'>Official documentation</a>",
+      
+       input_fields: lambda do |object_definitions|
+        [
+          {
+            name: "external_id",
+            label: "External ID",
+            hint: "Provide the External ID of the Asset that you want to cancel.",
+            optional: false
+          },
+          {
+            name: "external_uid",
+            label: "External UID",
+            hint: "Provide the External UID of the Asset that you want to cancel.",
+            optional: false
+          },
+          
+        ]
+      end,
+      
+      execute: lambda do |connection, input|
+        
+        params = {
+          "type": "cancel",
+          "asset": {
+            "external_id": "#{input["external_id"]}",
+            "external_uid": "#{input["external_uid"]}"
+          }
+        }
+   
+        post("https://api.connect.cloudblue.com/public/v1/requests", params).
```

Comment:
it should be 
```suggestion
        post("/public/v1/requests", params).
```
workato to construct the initial part of URL using base-uri method